### PR TITLE
feat(claim.go): add Outputs field to Claim struct

### PR DESCRIPTION
**Pull request by vdice**
- add Outputs field to Claim struct, per cnab-spec (updated in https://github.com/deislabs/cnab-spec/pull/217)

### CODE REVIEW

File: claim/claimstore_test.go [hunk 1]
```diff
@@ -118,3 +118,49 @@ func TestReadAll(t *testing.T) {
...
+	is.NoError(err, "Failed to store claim")
+
+	c, err := store.Read("foo")
+	is.NoError(err, "Failed to read claim")
+
+	want := map[string]interface{}{
+		"foo-output": true,
+		"bar-output": "bar",
+	}
+	is.Equal(c.Outputs, want, "Wrong outputs on claim")
```
Comment (carolynvs) [inline claim/claimstore_test.go]: The signature for Equal is first the expected result, then the actual result. This got switched here which can lead to confusing error messages if the test were to fail.

File: claim/claimstore_test.go [hunk 2]
```diff
@@ -118,3 +118,49 @@ func TestReadAll(t *testing.T) {
...
+	is.NoError(err, "Failed to store claim")
+
+	c, err = store.Read("foo")
+	is.NoError(err, "Failed to read claim")
+
+	want = map[string]interface{}{
+		"foo-output": true,
+		"bar-output": "baz",
+	}
+	is.Equal(c.Outputs, want, "Wrong outputs on claim")
```
Comment (carolynvs) [inline claim/claimstore_test.go]: Same here, the parameter order should be reversed.

File: claim/claimstore_test.go (hunk 1 above)
Comment (vdice) [reply claim/claimstore_test.go]: Thank you!  Fixed.

File: claim/claimstore_test.go (hunk 2 above)
Comment (vdice) [reply claim/claimstore_test.go]: Thank you!  Fixed.

**Pull request merged**